nixos/modules/virtualisation/google-compute-config.nix

```diff
@@ -2,13 +2,55 @@
 with lib;
 let
   gce = pkgs.google-compute-engine;
+  agent = pkgs.google-compute-guest-agent;
+  guest-configs = pkgs.google-compute-configs;
 in
 {
   imports = [
     ../profiles/headless.nix
     ../profiles/qemu-guest.nix
   ];
 
+  environment.etc."sysctl.d/11-gce-network-security.conf".source = "${gce}/sysctl.d/11-gce-network-security.conf";
+
+  environment.etc."default/instance_configs.cfg".text = ''
+[Accounts]
+deprovision_remove = false
+gpasswd_add_cmd = gpasswd -a {user} {group}
+gpasswd_remove_cmd = gpasswd -d {user} {group}
+groupadd_cmd = groupadd {group}
+useradd_cmd = useradd -m -s ${pkgs.bash}/bin/bash -p * {user}
+userdel_cmd = userdel -r {user}
+
+[Daemons]
+accounts_daemon = false
+clock_skew_daemon = true
+network_daemon = true
+oslogin_daemon = false
+
+[InstanceSetup]
+host_key_types = ecdsa,ed25519,rsa
+network_enabled = true
+optimize_local_ssd = false
+set_boto_config = true
+set_host_keys = false
+set_multiqueue = true
+
+[IpForwarding]
+ethernet_proto_id = 66
+ip_aliases = false
+target_instance_ips = false
+
+[MetadataScripts]
+default_shell = ${pkgs.bash}/bin/bash
+run_dir =
+shutdown = true
+startup = true
+
+[NetworkInterfaces]
+dhcp_command =
+ip_forwarding = false
+setup = false'';
```

Comment:
```suggestion
    setup = false
  '';
```

Reply:
This will add a undesirable empty line.

```
~ » cat /tmp/test.nix
''
  hello
  1 newline too many
''
~ » nix eval -f /tmp/test.nix
"hello\n1 newline too many\n"

~ » cat /tmp/test.nix
''
  hello
  better :)''

~ » nix eval -f /tmp/test.nix
"hello\nbetter :)"
```